src/backend/core/api/viewsets.py

```diff
@@ -368,15 +367,15 @@ class DocumentAccessViewSet(
 
     POST /api/v1.0/documents/<resource_id>/accesses/ with expected data:
         - user: str
-        - role: str [owner|admin|member]
+        - role: str [admin|editor|reader]
```

Comment:
Seems to be `administrator`.
```suggestion
        - role: str [administrator|editor|reader]
```

Reply:
done!